### PR TITLE
directory for log files

### PR DESCRIPTION
**Pull request by LisaHollstein**
I moved the files `empty_heattree_samples.txt`, `no_species_found_for_chromosomes.csv` and `no_genus_found for chromosomes.csv` to a new `logs/` directory.

No extra scripts are needed for this.

I am also thinking about moving `excluded_taxa.csv` to `logs/`: @colindaven what do you think about that?

### CODE REVIEW

**Comment by colindaven** [comment]
Hi Lisa, yes, pls move `excluded_taxa.csv` to `logs/`

File: run_haybaler.sh [hunk 1]
```diff
@@ -61,3 +61,10 @@ fi
 python3 haybaler.py -i "$input_files" -p . -op $outputDir  -o haybaler.csv
 # for pipeline testing only!!
 #python3 haybaler.py -i "$input_files" -p . -op $outputDir  -o haybaler.csv --readcount_limit 1 --rpmm_limit 10
+
+# Move log file into log directory
+if [[ ! -d $outputDir/logs ]]
+    then
+    mkdir $outputDir/logs
```
Comment (colindaven) [inline run_haybaler.sh:68]: You can shorten this whole thing to `mkdir -p bla/logs` - I only learned this recently

Comment (LisaHollstein) [reply run_haybaler.sh]: Thanks, I changed it for the other (not log directories) in run `run_heattrees.sh` as well

**Review by colindaven — APPROVED** [review]
Thanks for that

**Pull request merged**